.eslintrc.js

```diff
@@ -119,5 +119,13 @@ module.exports = {
         ...defaultRules,
       },
     },
+    {
+      files: ['**/?(*.)+(spec).[jt]s?(x)'],
+      excludedFiles: ['**/cypress/**/*.[jt]s?(x)'],
+      extends: [...defaultExtends, 'plugin:cypress/recommended'],
+      rules: {
+        ...defaultRules,
+      },
+    },
```

Comment:
Invalid jest errors were being displayed when writing the Cypress tests. Installed the cypress eslint rules and applying to all cypress specific tests now.